useful-scripts/importer-tests/importer.py

```diff
@@ -0,0 +1,217 @@
+import os
+import time
+import json
+import requests
+import argparse
+import csv
+import urllib3
+import datetime
+import random 
+import sys
+from pathlib import Path
+from dotenv import load_dotenv
+import zipfile
+import glob 
+from os.path import basename
+
+""" 
+Instructions
+
+- place a zip file (containing a data and a meta file) in an archive file called 'archive.zip' in the root of ~/importer-tests
+
+- move the .env.example to .env ('mv .env.example .env')
+
+- get a jwt token from Admin EES & place in .env file
+
+- get the topic ID from the &topicid query parameter on Admin EES 
+
+- place the API url value in the .env file (i.e. https://localhost:5021, dev Admin URL etc.). Do not leave a leading slash at the end of this URL. 
+
+- Command to run importer script: pipenv run python importer.py 
+
+- use the -c option if you want to import more than one subject (i.e. 'pipenv run python importer.py -c 5')
+""" 
+
+parser = argparse.ArgumentParser(prog="pipenv run python importer.py", description="Used to load test importer")
+
+parser.add_argument("-c", "--count",
+                    dest="count",
+                    default=1,
+                    type=int,
+                    help="num of subjects to upload")
+
+args = parser.parse_args()
+
+
+def create_publication(api_url):
+    run_identifier = datetime.datetime.utcnow().strftime('%Y%m%d-%H%M%S')
+    print(f'Starting load test with RUN_IDENTIFIER: {run_identifier}')
+    response = requests.request(
+        "POST",
+        url=f'{api_url}/api/publications',
+        headers={
+            "Content-Type": "application/json",
+            "Authorization": f"Bearer {jwt_token}",
```

Comment:
You'll need to pass this in as a parameter to this method I think

Reply:
Don't think the jwt_token needs to be passed in as a parameter as it'll be available as a global variable (due to the declaration being under `if __name__ == "__main__":`